workflow-ui/core-android/src/androidTest/java/com/squareup/workflow1/ui/container/BackStackContainerPersistenceTest.kt

```diff
@@ -93,7 +93,7 @@ internal class BackStackContainerPersistenceTest {
     scenario.onActivity {
       assertThat(it.consumeLifecycleEvents())
         .containsExactly(
-          "nested onViewCreated viewState=",
+          "onViewCreated viewState=",
```

Comment:
Can you add a note on the reason for this change? It's not clear to me.

Reply:
`ScreenViewFactory.buildView` doesn't have access to the rendering. That's actually a pretty big change that I should call out more explicitly. My reasoning was that I should make the method as small as possible, and try not to add it back unless I bumped into an existing case that couldn't be accomodated. So far I haven't (outside of this test).

I'm still a bit uncertain about the choice. I like the fact that it removes any ambiguity about whether `buildView` is expected to call `updateView` itself -- can't do that if you can't see the rendering.

Reply:
I've updated the commit message to call this out, and also to discuss why `ScreenViewFactory` does not get to answer `canShow`.